ROUTE53: Initial AWS European Sovereign Cloud support (#4342)
StackExchange/dnscontrol: documentation/provider/route53.md

You can find some other ways to authenticate to Route53 in the [go sdk configuration](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html).

## AWS European Sovereign Cloud (aws.eu)

> You can use AWS European Sovereign Cloud API endpoints by defining "eusc-de-east-1" to Region variable in `creds.json`. By default global AWS uses region "us-east-1" and you don't need to define it explicitly.

Reviewer: ```suggestion
You can use AWS European Sovereign Cloud API endpoints by setting the `Region` variable to `eusc-de-east-1` in `creds.json`. By default global AWS uses region "us-east-1" if not set explicitly.
```
